data/langdb.yaml

```diff
@@ -493,6 +493,7 @@ languages:
   tl: [Latn, [AS], Tagalog]
  # A very complicated case. Names.php is Cyrl. In TWN they argue about Cyrl, Latn, and Arab. I can't find reliable external sources. --Amir
   tly: [Cyrl, [EU, AS, ME], толышә зывон]
+  tmr: [Hebr, [ME, EU, AM], Jewish Babylonian Aramaic]
```

Comment:
I could not find any reference that says this is used in AM, @amire80  can you verify the regions here?

@hhm0 After adding this file, you need to run data/ ulsdata2json.php to generate json and js code from this YAML file